compiler/include/expr.h

```diff
@@ -381,13 +381,11 @@ static inline bool isAliveQuick(Symbol* symbol) {
 }
 
 static inline bool isAlive(Symbol* symbol) {
-  if (symbol->hasFlag(FLAG_GLOBAL_TYPE_SYMBOL)) return true;
-  if (! symbol->defPoint) return false;
-  return isAliveQuick(symbol);
+  return symbol->defPoint && isAlive(symbol->defPoint);
```

Comment:
Do you know why the case for FLAG_GLOBAL_TYPE_SYMBOL was added? Do you know that we can remove it? Certainly good testing results is one indication.
Also, can we now remove isAliveQuick ? It does not seem to be any "quicker" than isAlive at this point.